lib/commands/general.js

```diff
@@ -216,6 +221,100 @@ commands.closeApp = async function () {
   await this.adb.forceStop(this.opts.appPackage);
 };
 
+commands.getDisplayDensity = async function () {
+  let out = await this.adb.shell(['getprop', 'ro.sf.lcd_density']);
+  if (out) {
+    return parseInt(out);
```

Comment:
Nvm, I think I'll just debug log it and let it flow down to the last `log.errorAndThrow` in the case of NaNs.